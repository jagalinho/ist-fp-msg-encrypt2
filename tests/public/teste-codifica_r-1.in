resul = codifica_r(faz_pos(0, 1), faz_pos(3, 3))
resul
