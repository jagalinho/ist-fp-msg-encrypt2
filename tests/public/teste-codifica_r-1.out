(faz_pos(0, 3),  faz_pos(3, 1))
